FRAuth/FRAuth/SocialLogin/AppleSignInHandler.swift

```diff
@@ -115,12 +125,19 @@ extension AppleSignInHandler: ASAuthorizationControllerDelegate {
         switch authorization.credential {
         case let appleIDCredential as ASAuthorizationAppleIDCredential:
             FRLog.v("ASAuthorizationAppleIDCredential received: \(appleIDCredential)")
-            guard let tokenData = appleIDCredential.identityToken, let idToken = String(data: tokenData, encoding: .utf8) else {
+                      
+            let appleSignInResponse = AppleSignInResponse(appleIDCredential)
+            guard let authorization_code = appleSignInResponse.code, let id_token = appleSignInResponse.id_token, let IDToken1tokenJSON = try? JSONEncoder().encode(appleSignInResponse), let IDToken1token = String(data: IDToken1tokenJSON, encoding: .utf8) else {
                 self.completionCallback?(nil, nil, SocialLoginError.unsupportedCredentials("Failed to parse received credentials data (ASAuthorizationAppleIDCredential.identityToken)"))
                 return
             }
             
```

Comment:
What are the chances that the above code might result in an early return when _acceptsJSON = false_ and we only need the _id_token_ ? I'm just making sure we don't break anything for the older versions here by adding all this checks and trying to create a JSON when we don't really need it.